README.md

```diff
@@ -10,8 +10,9 @@ _How do I, as a developer, start using Taplytics?_
 
 1. _Sign up for a free account at [Taplytics.com](https://taplytics.com?utm_source=github&utm_campaign=documentation&utm_medium=content)._
 2. _Install the SDK. Steps [here](/START.md)._
-3. Create [Experiments](/EXPERIMENTS.md) or send [Push Notifications](/PUSH.md) to your users!
+3. _Install the SDK Using Segment [here](/SEGMENT.md)_
```

Comment:
No this seems like its a required step. Just put it in it's own 'SEGMENT' section again and just link to the file from there.